oozie-to-airflow/tests/mappers/test_decision_mapper.py

```diff
@@ -50,14 +48,17 @@ def test_convert_to_text(self):
         # TODO
         # decision mapper does not have the required EL parsing to correctly get
         # parsed, so once that is finished need to redo tests.
-        mapper = decision_mapper.DecisionMapper(
-            oozie_node=self.decision_node, name="test_id", trigger_rule=TriggerRule.DUMMY
-        )
+        mapper = self._get_mapper()
         res = mapper.convert_to_text()
         ast.parse(res)
 
-    # pylint: disable=no-self-use
     def test_required_imports(self):
-        imps = decision_mapper.DecisionMapper.required_imports()
+        mapper = self._get_mapper()
+        imps = mapper.required_imports()
         imp_str = "\n".join(imps)
         ast.parse(imp_str)
+
+    def _get_mapper(self):
+        return decision_mapper.DecisionMapper(
+            oozie_node=self.decision_node, name="test_id", trigger_rule=TriggerRule.DUMMY
```

Comment:
Parameter `trigger_rule` have a default value - `TriggerRule.Dummy`. Why are you setting it again?

Reply:
Can you point me to where this default value is?

From what I can find the decision mapper's default value is `TriggerRule.ALL_DONE`:
```
class DecisionMapper(BaseMapper):
(...)
    def __init__(
        self,
        oozie_node: Element,
        name: str,
        trigger_rule: str = TriggerRule.ALL_DONE,
        params: Dict[str, str] = None,
    ):
```

If the explicit trigger_rule setting here is removed, the unit tests are failing.

Reply:
You're right. This is important in this particular place, but in real code it does not matter. This class is used only once in the project.
https://github.com/GoogleCloudPlatform/cloud-composer/blob/fix-subworkflow-fixed-name/oozie-to-airflow/converter/parser.py#L177
In this case, the value is not used and I think it is more important to test real cases.

Reply:
Besides, It does not matter much to me. I plan to delete this argument. 
https://github.com/GoogleCloudPlatform/cloud-composer/pull/146/files

I will give a green light if this problem is solved:
https://github.com/GoogleCloudPlatform/cloud-composer/pull/124#discussion_r279382911

Reply:
Yes, I plan on fixing it today.